resources/sts/4.12/hypershift/sts_hcp_support_permission_policy.json

```diff
@@ -1,341 +1,294 @@
 {
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-      "Effect": "Allow",
-      "Action": [
-        "cloudtrail:DescribeTrails",
-        "cloudtrail:LookupEvents",
-        "cloudwatch:GetMetricData",
-        "cloudwatch:GetMetricStatistics",
-        "cloudwatch:ListMetrics",
-        "ec2-instance-connect:SendSerialConsoleSSHPublicKey",
-        "ec2:Describe*",
-        "ec2:GetAssociatedIpv6PoolCidrs",
-        "ec2:GetSerialConsoleAccessStatus",
-        "ec2:GetTransitGatewayAttachmentPropagations",
-        "ec2:GetTransitGatewayMulticastDomainAssociations",
-        "ec2:GetTransitGatewayPrefixListReferences",
-        "ec2:GetTransitGatewayRouteTableAssociations",
-        "ec2:GetTransitGatewayRouteTablePropagations",
-        "ec2:SearchLocalGatewayRoutes",
-        "ec2:SearchTransitGatewayMulticastGroups",
-        "elasticloadbalancing:ConfigureHealthCheck",
-        "elasticloadbalancing:Describe*",
-        "iam:GetRole",
-        "iam:ListRoles",
-        "kms:CreateGrant",
-        "route53:GetHosted*",
-        "route53:List*",
-        "s3:GetBucketTagging",
-        "s3:GetObjectAcl",
-        "s3:GetObjectTagging",
-        "s3:ListAllMyBuckets",
-        "sts:DecodeAuthorizationMessage",
-        "tiros:CreateQuery",
-        "tiros:GetQuery*"
-      ],
-      "Resource": "*"
-    },
-    {
-      "Effect": "Allow",
-      "Action": [
-        "ec2:CreateTags"
-      ],
-      "Resource": [
-        "arn:aws:ec2:*:*:instance/*",
-        "arn:aws:ec2:*:*:volume/*",
-        "arn:aws:ec2:*:*:vpc/*",
-        "arn:aws:ec2:*:*:subnet/*"
-      ],
-      "Condition": {
-        "StringEquals": {
-          "ec2:CreateAction": [
-            "RunInstances",
-            "CreateVolume"
-          ]
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "cloudtrail:DescribeTrails",
+                "cloudtrail:LookupEvents",
+                "cloudwatch:GetMetricData",
+                "cloudwatch:GetMetricStatistics",
+                "cloudwatch:ListMetrics",
+                "ec2:Describe*",
+                "ec2:GetTransitGatewayAttachmentPropagations",
+                "ec2:GetTransitGatewayMulticastDomainAssociations",
+                "ec2:GetTransitGatewayPrefixListReferences",
+                "ec2:GetTransitGatewayRouteTableAssociations",
+                "ec2:GetTransitGatewayRouteTablePropagations",
+                "elasticloadbalancing:Describe*",
+                "iam:GetRole",
+                "iam:ListRoles",
+                "kms:CreateGrant",
```

Comment:
This should be scoped now once PR #1546 merges

Reply:
I have pulled these changes in due to merge conflicts.